integration/apidefinition_test.go

```diff
@@ -77,30 +77,20 @@ func TestApiDefinitionJSONSchemaValidation(t *testing.T) {
 				apiDef.Spec.VersionData.Versions = map[string]model.VersionInfo{
 					defaultVersion: {Name: defaultVersion, UseExtendedPaths: true, ExtendedPaths: eps},
 				}
-			}, envConf)
+			})
 			eval.NoErr(err) // failed to create apiDefinition
 
+			apiDef := v1alpha1.ApiDefinition{
```

Comment:
Sorry, I couldn't understand this part. Why we are creating an ApiDefinition on Tyk here?

Reply:
I need to pass that object to `waitForTykResourceCreation`

Reply:
okay 👍  
instead of calling `waitForTykResourceCreation` here, shouldn't creating a resource in k8s create a resource in Tyk? because previously, it seems like we were not creating resources on Tyk manually.

Reply:
Sometimes it takes time to create resource on Tyk. Because of which successive operations fails.
That's why I have added `waitForTykResourceCreation` in situations we want to make sure API is created on Tyk before proceeding.

Reply:
👍 thanks for the explanation